platform/mellanox/docker-syncd-mlnx-rpc.mk

```diff
@@ -20,7 +20,7 @@ SONIC_INSTALL_DOCKER_IMAGES += $(DOCKER_SYNCD_MLNX_RPC)
 endif
 
 $(DOCKER_SYNCD_MLNX_RPC)_CONTAINER_NAME = syncd
-$(DOCKER_SYNCD_MLNX_RPC)_RUN_OPT += --net=host --privileged -t
```

Comment:
@stephenxs why do we need to remove --net=host ?

Reply:
There is a duplicate --net=host [here](https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/docker_image_ctl.j2#L211). So this one has to be removed.